qemu/tests/chardev_acpi.py

```diff
@@ -0,0 +1,34 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    acpi description of serial and parallel ports incorrect
+    with -chardev/-device:
+    1) Boot guest A with isa-serial with tty chardev backend
+    2) Enter into guest A
+        udevadm info --query path --name /dev/ttyS0
+        !! --attribute-walk | grep looking
+    3) Boot guest B with -serial /dev/ttyS0
+    4) repeat step 2 for guest B
+    5) Check if the result are same for A and B.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    output = []
+    p = 'udevadm info --query path --name /dev/ttyS0'
+    for x in range(2):
+        vm = env.get_vm("vm{}".format(x+1))
+        if x >= 1:
```

Comment:
Do we need this judgment branch?

Reply:
Hi yes, we are comparing the results from VM with different serial parameter